pages/1.11/deploying-services/private-docker-registry/index.md

```diff
@@ -129,13 +129,13 @@ Follow these steps to add your Docker registry credentials to the [DC/OS Enterpr
    **Note:** As of DC/OS version 1.10.0, you can add a file to the secret store only using the DC/OS CLI.
 
    ```bash
-   dcos security secrets create --value-file=config.json <path/to/secret>
+   dcos security secrets create -f config.json <path/to/secret>
```

Comment:
Let's stick with the long form - much more explanatory.

Reply:
✅